.travis.yml

```diff
@@ -17,24 +18,24 @@ deploy:
   - provider: script
     skip_cleanup: true
     script:
-      npx semantic-release
+      npm run semantic-release
   # deploy develop to Staging
   - provider: script
     script:
-      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --exclude "*" --include "*.html" --content-type "text/html; charset=utf-8" &&
-      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --exclude "*" --include "*.css" --content-type "text/css; charset=utf-8" &&
-      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --exclude "*" --include "*.js" --content-type "application/javascript; charset=utf-8" &&
-      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --include "*" --exclude "*.js" --exclude "*.html" --exclude "*.css"
+      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --exclude "*" --include "packages/gravity-ui-web/*.html" --content-type "text/html; charset=utf-8" &&
+      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --exclude "*" --include "packages/gravity-ui-web/*.css" --content-type "text/css; charset=utf-8" &&
+      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --exclude "*" --include "packages/gravity-ui-web/*.js" --content-type "application/javascript; charset=utf-8" &&
+      ~/.local/bin/aws s3 sync dist s3://${STAGING_BUCKET} --delete --region=${STAGING_BUCKET_REGION} --include "*" --exclude "packages/gravity-ui-web/*.js" --exclude "*.html" --exclude "*.css"
```

Comment:
Hmm... upon closer inspection, are these right?

Is it not `packages/gravity-ui-web/dist` that needs to be synced now, rather than just `dist`? I presume then don't need the path later on for the `--include` and `--exclude` bits (I _assume_ those are relative to the dir being synced).

So, putting it together, something like this:
```
 ~/.local/bin/aws s3 sync packages/gravity-ui-web/dist s3://${PROD_BUCKET} --delete --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.html" --content-type "text/html; charset=utf-8" &&
...
```

Apologies for not spotting this one sooner.

Reply:
Good catch here, looks like I edited the wrong path with the AWS commands.